pkg/operator/webhooksupportabilitycontroller/degraded_webhook_conversion.go

```diff
@@ -48,5 +56,24 @@ func hasCRDConversionWebhookConfiguration(crd *apiextensionsv1.CustomResourceDef
 	if clientConfig == nil || clientConfig.Service == nil {
 		return false
 	}
+	if waitForServiceCABundle(crd.Annotations, clientConfig.CABundle) {
+		return false
+	}
 	return true
 }
+
+// waitForServiceCABundle returns true if a resource has the service-ca
+// annotation and the caBundle hasn't been yet provided.
+func waitForServiceCABundle(annotations map[string]string, caBundle []byte) bool {
+	if hasServiceCaAnnotation(annotations) {
```

Comment:
do we need to check for the annotation, shouldn't it be enough to just check for the length of the caBundle?

Reply:
I think we do, at least when the resource is annotated, in that case we rely on the service-ca to provide the bundle.

Reply:
In general, a caBundle is optional and when it is empty we use CAs provided by the underlying OS.